go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -166,17 +166,19 @@ func (ef *egressFirewall) addLogicalRouterPolicyToClusterRouter(hashedAddressSet
 			return fmt.Errorf("error rule.to.cidrSelector %s is not a valid CIDR (%+v)", rule.to.cidrSelector, err)
 		}
 		if !utilnet.IsIPv6(ipAddress) {
-			match = fmt.Sprintf("match=\"ip4.dst == %s && ip4.src == $%s\"", rule.to.cidrSelector, hashedAddressSetNameIPv4)
+			match = fmt.Sprintf("match=\"ip4.dst == %s && ip4.src == $%s", rule.to.cidrSelector, hashedAddressSetNameIPv4)
```

Comment:
do you need the match= here? I think you can just put the match criteria like: https://github.com/ovn-org/ovn-kubernetes/pull/1720/files#diff-04a3ee61bce76dbc7d0f583289d5f608R514

and use the lr-policy-add command or am I missing something?

Reply:
using `lr-policy-add` does not allow to tag the `logical-router-policy` with an externalID which makes deletion of specific policies much easier